backend/src/main/java/com/dhbw/tutorsystem/offer/Offer.java

```diff
@@ -0,0 +1,35 @@
+package com.dhbw.tutorsystem.offer;
+import java.time.LocalDate;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name="offer") 
+public class Offer {
```

Comment:
Might want to consider specifying `Offer` and `Request` more precisely, such as `TutorialOffer`and `TutorialRequest`

Reply:
Sounds reasonable

Reply:
Was solved in 09524b86d247fef36caf37bb4e3d0e135a82737b